<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="1" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-01T18:33:11 IST" name="Suite" finished-at="2021-09-01T18:33:57 IST" duration-ms="46160">
    <groups>
    </groups>
    <test started-at="2021-09-01T18:33:11 IST" name="Test" finished-at="2021-09-01T18:33:57 IST" duration-ms="46160">
      <class name="com.StromgerMe.Scripts.TC_lg_001">
        <test-method is-config="true" signature="configBC()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:11 IST" name="configBC" finished-at="2021-09-01T18:33:13 IST" duration-ms="1904" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBC -->
        <test-method is-config="true" signature="configBM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:14 IST" name="configBM" finished-at="2021-09-01T18:33:15 IST" duration-ms="1657" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBM -->
        <test-method signature="LoginTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:15 IST" name="LoginTest" data-provider="LoginData" finished-at="2021-09-01T18:33:26 IST" duration-ms="10682" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC_lg_001]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[mngr349955]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[vEsYpyp]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="configAM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:26 IST" name="configAM" finished-at="2021-09-01T18:33:26 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAM -->
        <test-method is-config="true" signature="configBM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:26 IST" name="configBM" finished-at="2021-09-01T18:33:27 IST" duration-ms="1204" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBM -->
        <test-method signature="LoginTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:27 IST" name="LoginTest" data-provider="LoginData" finished-at="2021-09-01T18:33:34 IST" duration-ms="6303" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC_lg_001]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin1]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[admin1]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.StromgerMe.Scripts.TC_lg_001.LoginTest(TC_lg_001.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="configAM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:36 IST" name="configAM" finished-at="2021-09-01T18:33:36 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAM -->
        <test-method is-config="true" signature="configBM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:36 IST" name="configBM" finished-at="2021-09-01T18:33:38 IST" duration-ms="1678" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBM -->
        <test-method signature="LoginTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:38 IST" name="LoginTest" data-provider="LoginData" finished-at="2021-09-01T18:33:44 IST" duration-ms="6349" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC_lg_001]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin12 ]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[admin12]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.StromgerMe.Scripts.TC_lg_001.LoginTest(TC_lg_001.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="configAM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:46 IST" name="configAM" finished-at="2021-09-01T18:33:46 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAM -->
        <test-method is-config="true" signature="configBM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:46 IST" name="configBM" finished-at="2021-09-01T18:33:48 IST" duration-ms="1366" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBM -->
        <test-method signature="LoginTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:48 IST" name="LoginTest" data-provider="LoginData" finished-at="2021-09-01T18:33:54 IST" duration-ms="6551" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[TC_lg_001]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[admin12]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Invalid]]>
              </value>
            </param>
          </params>
          <exception class="junit.framework.AssertionFailedError">
            <full-stacktrace>
              <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.StromgerMe.Scripts.TC_lg_001.LoginTest(TC_lg_001.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- junit.framework.AssertionFailedError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="configAM()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:57 IST" name="configAM" finished-at="2021-09-01T18:33:57 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAM -->
        <test-method is-config="true" signature="cofigAC()[pri:0, instance:com.StromgerMe.Scripts.TC_lg_001@619713e5]" started-at="2021-09-01T18:33:57 IST" name="cofigAC" finished-at="2021-09-01T18:33:57 IST" duration-ms="462" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cofigAC -->
      </class> <!-- com.StromgerMe.Scripts.TC_lg_001 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
